querydsl-jdo/src/test/java/com/querydsl/jdo/BasicsTest.java

```diff
@@ -93,10 +94,14 @@ public void list_distinct() {
 
     @Test
     public void list_distinct_two_sources() {
-        // XXX List implementation of JDO provider has weird equals implementation
-        assertEquals(
-                ImmutableList.copyOf(query().from(product, product2).select(product, product2).fetch()),
-                ImmutableList.copyOf(query().from(product, product2).distinct().select(product, product2).fetch()));
+        try {
+            // XXX List implementation of JDO provider has weird equals implementation
```

Comment:
Do you think so? Do you mean copying it over into a new list?
I don't quite know if it's worth the overhead, personally.
What's your view on this?